app/scripts/dimApp.i18n.js

```diff
@@ -16,7 +16,15 @@
           Equip: "Equip",
           Vault: "Vault",
           Vanguard: "Vanguard",
-          Reputation: "Reputation"
+          Reputation: "Reputation",
+          Super_cooldown: "Super cooldown",
+          Melee_cooldown: "Melee cooldown",
+          Grenade_cooldown: "Grenade cooldown",
+          tier_progress: "{{progress}} for {{tier}}",
+          About: "About",
+          'Support DIM': "Support DIM",
```

Comment:
should we have consistency with our keys?
`support_dim`?

some are also not all lower case... not a big deal, but it could make things easier?